packages/sx.js/src/clients/starknet/ethereum-tx/index.ts

```diff
@@ -223,7 +223,7 @@ export class EthereumTx {
 
   async estimateProposeFee(signer: Signer, data: Propose) {
     const address = await signer.getAddress();
-    const hash = await this.getProposeHash(signer, data);
+    const hash = await this.getProposeHash(address, data);
```

Comment:
Can we extract those sx.js changes to separate PR? With `yarn changeset` to generate changelog for it.

Reply:
https://github.com/snapshot-labs/sx-monorepo/pull/969